cax/tasks/data_mover.py

```diff
@@ -190,8 +190,15 @@ def copySCP(self, datum_original, datum_destination, server, username, option_ty
         client.close()
 
     def each_run(self):
-        for data_type in ['raw', 'processed']:
-            self.log.debug("%s" % data_type)
+        print("got to each run")
+        try:
```

Comment:
Don't try except.  Check if 'data_types' in get_config(), then data_types = blah with a data_types = [a,b,] before.